server.ts

```diff
@@ -170,11 +182,17 @@ app.post("/conferences", async (req, res) => {
     const slug = slugify(name).toLowerCase();
     console.log("using slug", slug);
     if (slugsToIds.has(slug)) {
-      res.status(409).send();
-      return;
+      let conferenceId = slugsToIds.get(slug)!;
+      try {
+        await getConference(conferenceId);
```

Comment:
why does the sample app need to attempt to get the conference from our service if it doesn't use the response?  Just send the 409 CONFLICT response if the slug already exists in the map?